<launch>
    <node name="usb_cam_image_py" pkg="transbot_visual" type="usb_cam_image.py" output="screen"/>
<!--    <node name="astra_rgb_image_cpp" pkg="transbot_visual" type="get_color_image" output="screen"/>-->
</launch>
